source/docs/software/driverstation/driver-station-errors-warnings.rst

```diff
@@ -52,11 +52,11 @@ Radio Detection Times
 
 .. code-block::
 
-  WARNING<Code>44008 occured at FRC_NetworkCommunications
+  WARNING<Code>44008 occurred at FRC_NetworkCommunications
```

Comment:
Is this misspelled in the actual error? If so then we need to leave it and ask NI to update the spelling on their end.

Reply:
I do not know, and as an FTC student I unfortunately can't check easily. Would it be possible for someone else to check?

Reply:
I can take a look later this week unless someone beats me to it.

Reply:
Someone quoted it in a CD post with the correct spelling, so either they ran spell check on it, or NI fixed it